Server/db/models/HardwareCheck.js

```diff
@@ -64,5 +64,7 @@ const HardwareCheckSchema = mongoose.Schema(
 );
 
 HardwareCheckSchema.index({ createdAt: 1 });
+HardwareCheckSchema.index({ monitorId: 1, createdAt: 1 });
+HardwareCheckSchema.index({ monitorId: 1, createdAt: -1 });
```

Comment:
Line 67 and 68 will be needed both from client (ascending and descending createdAt)?  so most recent and oldest will be queried in one client request in order to show the details, for past 24 hours and historical data , I see, 
this will improve performance when query for most recent and oldest (such as historical data)